servlet/pom.xml

```diff
@@ -31,8 +31,7 @@
     <groupId>io.undertow</groupId>
     <artifactId>undertow-servlet</artifactId>
     <version>2.2.21.Final-SNAPSHOT</version>
-
-    <name>Undertow Servlet</name>
+    <name>Undertow Servlet - Java EE Variant</name>
```

Comment:
In the same way, I don't think we should add Java EE variant here. I think we could state something about this on the README file instead

Reply:
Fixed @fl4via